docs/sources/shared/datasources/tempo-traces-to-profiles.md

```diff
@@ -21,49 +21,59 @@ Using Trace to profiles, you can use Grafana’s ability to correlate different
 **Trace to profiles** lets you link your Grafana Pyroscope data source to tracing data.
 When configured, this connection lets you run queries from a trace span into the profile data.
 
+{{< youtube id="AG8VzfFMLxo" >}}
+
 There are two ways to configure the trace to profiles feature:
 
 - Use a simplified configuration with default query, or
 - Configure a custom query where you can use a template language to interpolate variables from the trace or span.
 
 To use trace to profiles, navigate to **Explore** and query a trace. Each span now links to your queries. Clicking a link runs the query in a split panel. If tags are configured, Grafana dynamically inserts the span attribute values into the query. The query runs over the time range of the (span start time - 60) to (span end time + 60 seconds).
 
-![Selecting a link in the span queries the profile data source](/static/img/docs/tempo/profiles/tempo-profiles-Span-link-profile-data-source.png)
+![Selecting a link in the span queries the profile data source](/media/docs/tempo/profiles/tempo-trace-to-profile.png)
+
+To use trace to profiles, you must have a configured Grafana Pyroscope data source. For more information, refer to the [Grafana Pyroscope data source](/docs/grafana/latest/datasources/grafana-pyroscope/) documentation.
 
-To use trace to profiles, you must have a configured Grafana Pyroscope data source. For more information, refer to the [Grafana Pyroscope data source documentation](/docs/grafana/latest/datasources/grafana-pyroscope/).
+**Embedded flame graphs** are also inserted into each span details section that has a linked profile (requires a configured Grafana Pyroscope data source).
+This lets you see resource consumption in a flame graph visualization for each span without having to navigate away from the current view.
+Hover over a particular block in the flame graph to see more details about the resources being consumed.
 
-## Use a simple configuration
+## Use a basic configuration
 
 To use a simple configuration, follow these steps:
 
 1. Select a Pyroscope data source from the **Data source** drop-down.
 1. Optional: Choose any tags to use in the query. If left blank, the default values of `service.name` and `service.namespace` are used.
 
-   The tags you configure must be present in the spans attributes or resources for a trace to profiles span link to appear. You can optionally configure a new name for the tag. This is useful for example if the tag has dots in the name and the target data source doesn't allow using dots in labels. In that case you can for example remap `service.name` to `service_name`.
+   The tags you configure must be present in the spans attributes or resources for a trace to profiles span link to appear.
```

Comment:
I don't know the terminology here... is `a trace to profiles span link` a name for the kind of link? or is a word missing here?

Reply:
So the feature name is "trace to profiles", which makes me wonder if it should be "traces-to-profiles span link" since traces-to-spans is functioning as an adjective.

Reply:
The hyphens def help :-)

Reply:
Okay!
```suggestion
   The tags you configure must be present in the spans attributes or resources for a trace-to-profiles span link to appear.
```